utilities/fault_injection_fs.cc

```diff
@@ -16,6 +16,8 @@
 
 #include "utilities/fault_injection_fs.h"
 
+#include <sys/types.h>
```

Comment:
Seems odd. Not sure how this got in here, and maybe not portable

Reply:
make format somehow did that hmmm let me check

Reply:
Not `make format` but VSCode probably

Reply:
Fixed